repository_service_tuf/cli/generate_key.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2022-2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+from rich import prompt
+from rich.console import Console  # type: ignore
+from securesystemslib.interface import (  # type: ignore
+    generate_and_write_ed25519_keypair_with_prompt,
+    generate_and_write_ecdsa_keypair_with_prompt,
+    generate_and_write_rsa_keypair_with_prompt,
+)
+
+from repository_service_tuf.cli import rstuf
+from repository_service_tuf.enums import KeyType
+
+
+console = Console()
+
+
+@rstuf.command()
+def generate_key() -> None:
+    """Generate cryptographic keys using the `securesystemslib` library"""
+
+    key_type: str = prompt.Prompt.ask(
+        f"\nChoose key type",
+        choices=KeyType.get_all_members(),
+        default=KeyType.KEY_TYPE_ED25519.value,
+    )
+
+    filepath: str = prompt.Prompt.ask(
+        "Enter the keys' [green]filename[/] (default path is the current "
+        "working directory) to store the files",
+        default="./rstuf-key",
+    )
+
+    match key_type:
+        case KeyType.KEY_TYPE_ED25519.value:
+            generate_and_write_ed25519_keypair_with_prompt(filepath)
+
+        case KeyType.KEY_TYPE_ECDSA.value:
+            generate_and_write_ecdsa_keypair_with_prompt(filepath)
+
+        case KeyType.KEY_TYPE_RSA.value:
+            generate_and_write_rsa_keypair_with_prompt(filepath)
```

Comment:
Hey @KAUTH,

We shouldn't use the `_with_prompt` from securesystemslib.
The reason is that we use encrypted keys always in the Worker/CLI Ceremony for now.
So, securesystemslib makes it optional, whereas in our use case is mandatory.